RosettaCore/src/main/java/com/hubspot/rosetta/internal/StoredAsJsonSerializer.java

```diff
@@ -44,4 +50,53 @@ public void acceptJsonFormatVisitor(JsonFormatVisitorWrapper visitor, JavaType t
       throw new RuntimeException(e);
     }
   }
+
+  @Override
+  public JsonSerializer<?> createContextual(SerializerProvider prov, BeanProperty property) throws JsonMappingException {
+    if (property == null) {
+      return this;
+    } else {
+      return new ContextualStoredAsJsonSerializer<>(_handledType, property);
+    }
+  }
+
+  private static class ContextualStoredAsJsonSerializer<T> extends NonTypedScalarSerializerBase<T> {
+    private final BeanProperty property;
+
+    ContextualStoredAsJsonSerializer(Class<T> t, BeanProperty property) {
+      super(t);
+      this.property = property;
+    }
+
+    @Override
+    public void serialize(T value, JsonGenerator gen, SerializerProvider provider) throws IOException {
+      ObjectMapper mapper = (ObjectMapper) gen.getCodec();
+      if (value == null) {
+        provider.defaultSerializeNull(gen);
+      }
+
+      JsonSerializer<Object> serializer = provider.findTypedValueSerializer(resolveRealType(value, mapper), false, null);
+      if (serializer != null) {
+        SegmentedStringWriter sw = new SegmentedStringWriter(new BufferRecycler());
+        JsonGenerator subGen = mapper.getFactory().createGenerator(sw);
+        try {
+          mapper.getSerializationConfig().initialize(subGen);
+          serializer.serialize(value, subGen, provider);
+        } finally {
+          subGen.close();
+        }
+        gen.writeString(sw.getAndClear());
+      } else {
+
+      }
```

Comment:
Empty else block?

Reply:
was intended to handle the map case, but became unnecessary